packages/core/src/routes/hook.ts

```diff
@@ -23,11 +23,19 @@ export default function hookRoutes<T extends AuthedRouter>(
 
   router.post(
     '/hooks',
-    koaGuard({ body: Hooks.createGuard.omit({ id: true }), response: Hooks.guard, status: 200 }),
+    koaGuard({
+      body: Hooks.createGuard
+        .omit({ id: true, event: true })
+        .required({ name: true, events: true }),
+      response: Hooks.guard,
+      status: [200, 422],
```

Comment:
where do we throw 422?

Reply:
Sorry, the old implementation has a 422, but now we need to remove it.